apps/earn-protocol-landing-page/components/layout/MasterPage/MasterPage.tsx

```
@@ -0,0 +1,34 @@
+import { type FC, type PropsWithChildren } from 'react'
+import { parseServerResponseToClient } from '@summerfi/app-utils'
+import Image from 'next/image'
+
+import systemConfigHandler from '@/server-handlers/system-config'
+
+import masterPageStyles from './MasterPage.module.scss'
+
+interface MasterPageProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)